packages/base-driver/lib/express/middleware.js

```diff
@@ -78,7 +78,9 @@ export function fixPythonContentType(basePath) {
  * @returns {any}
  */
 export function handleLogContext(req, res, next) {
-  const requestId = util.uuidV4();
+  const requestId = Array.isArray(req.headers['x-request-id'])
```

Comment:
Please update this expression to make it easier to read:

```js
/** @type {(string) => string | undefined} */
const fetchHeaderValue (name) => _.isArray(req.headers[name]) ? req.headers[name][0] : req.headers[name];

const requestId = fetchHeaderValue('x-request-id') || util.uuidV4();
....
const isSensitiveHeaderValue = fetchHeaderValue('x-appium-is-sensitive');
....
```

Reply:
Thank you for the feedback, I have addressed!